requirements.txt

```diff
@@ -2,3 +2,4 @@ recommonmark
 sphinx
 sphinx-markdown-tables
 sphinx-argparse
+visidata
```

Comment:
We are switching from instead of using a `visidata` submodule to installing it?

Reply:
This is for Sphinx when building the API docs. For some reason it throws more errors if visidata isn't installed.

And yes we did switch from using a submodule in #69. `./visidata` on master is a symlink to `node_modules/visidata`. I removed the symlink and switched to cloning the repo + checking out a specific commit instead, which satisfies the version pinning requirement and allowed me to get rid of the abandoned `napa` package.

Reply:
Ty!